api/preview/documentdb_types.go

```diff
@@ -40,11 +40,13 @@ type Resource struct {
 	PvcSize string `json:"pvcSize"`
 }
 
-type ClusterReplication struct {
-	// Primary is the name of the primary cluster for replication.
-	Primary string `json:"primary"`
-	// ClusterList is the list of clusters participating in replication.
-	ClusterList []string `json:"clusterList"`
+type PhysicalReplication struct {
+  // FleetEnabled determines whether to use KubeFleet mechanics for the replication
+	FleetEnabled bool     `json:"fleetEnabled,omitempty"`
```

Comment:
Should we say `enableFleetForCrossCloud`? as we dont need for replication in same cloud environment.